SayTheirNames/Source/Controller/Base/BaseNibLoading.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIViewController+STN.swift
+//  SayTheirNames
+//
+//  Created by evilpenguin on 5/31/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+public protocol BaseNibLoading {
```

Comment:
this file was removed in #39

Reply:
Let me merge with master again.

Reply:
didn't help 🤔  
would it be easier to re-apply the changes off the latest development, what do you think? if you do so, would be ideal to have renaming the project and adding SwiftLint as separate PRs

Reply:
I ended up removing the file. It was still hanging around in the folder but the project had no reference to it.